user/reference/bionic.md

```diff
@@ -0,0 +1,167 @@
+---
+title: The Ubuntu 18.04 Build Environment
+layout: en
+---
+
+## What This Guide Covers
+
+This guide provides an overview of the packages, tools and settings available in the Bionic environment.
+
+## Using Bionic
+
+To route your builds to Ubuntu 18.04 LTS, add the following to your `.travis.yml`:
+
+```yaml
+dist: bionic
+```
+{: data-file=".travis.yml"}
+
+Please note that Bionic is available on our hosted fully virtualized
+infrastructure. If you are running an Enterprise installation, please reach out
+to [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com?subject=Try%20out%20Bionic) to see how you can use the Bionic Docker images.
```

Comment:
@JaroslawSokol is Bionic on enterprise already available? Shall we have dedicated public instructions for it? Thank you!

Reply:
No, there is no support for Bionic available in last Enterprise release (2.2.11).
I think we should not mention it yet for Enterprise.
cc @danishkhan

Reply:
The note being removed